client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml

```diff
@@ -50,10 +50,11 @@
    <translation id="privateDatabase_dashboard_service_hosting_linked" qtlid="444456">Hébergement web lié</translation>
    <translation id="privateDatabase_dashboard_service_hosting_linked_none" qtlid="30646">Aucun</translation>
 
-   <translation id="privateDatabase_dashboard_ftp_informations_legacy" qtlid="74045">Connexion FTP</translation>
-   <translation id="privateDatabase_dashboard_ftp_informations_docker" qtlid="265470">Connexion SFTP</translation>
+   <translation id="privateDatabase_dashboard_connections_informations_title">Connexions</translation>
+   <translation id="privateDatabase_dashboard_ftp_informations_legacy">FTP</translation>
+   <translation id="privateDatabase_dashboard_ftp_informations_docker">SFTP</translation>
    <translation id="privateDatabase_dashboard_ftp_username" qtlid="56045">Nom d'utilisateur</translation>
-   <translation id="privateDatabase_dashboard_sql_informations" qtlid="263988">Connexion SQL</translation>
+   <translation id="privateDatabase_dashboard_sql_informations">SQL</translation>
```

Comment:
Wasn't the issue that we were issue the term `SQL` whereas it's a no SQL database ?